### ISSUE TITLE
GitHub Web Edit option

### ISSUE THREAD

**Issue by petrsx** [google/docsy#777]
Hi, GitHub now have "Web Editor" functionality if you press `.` on the file so you can edit it in VS code web instead of the simple `edit this file` in GitHub UI.

The idea is to an have configuration option `where` to edit page? This can be just a different link selection:

GitHub Edit url:

```
https://github.com/<USERNAME>/<REPOSITORY>/edit/main/quickstart/content/en/docs/_index.md
```

VS Code web url:

```
https://github.dev/<USERNAME>/<REPOSITORY>/blob/main/quickstart/content/en/docs/_index.md
```

Root url changed from `github.com` to `github.dev` and navigation from `edit` to `blob`

**Comment by LisaFC** [comment]
Ooh, just tried that and it's really nice! @chalin what do you think?

**Comment by petr-stupka** [comment]
I just copied my post from one of the chalin issue over, so it will not get forgotten since this is pretty cool feature from GitHub, agree

**Comment by chalin** [comment]
I haven't had time to try this again, but from what I recall, after being thrown into the editor and making changes, it didn't seem obvious how to submit a PR. That use case should be well supported / documented since folks will be editing a file with the intention of submitting a PR. The old interface makes that easy.

Regardless, we can still consider adding such a repo link.

**Comment by gwatts** [comment]
I agree with @chalin - you enter a far more complicated workflow if you land someone straight into vscode.  The current github editor makes things pretty straightforward if someone just wants to fix a typo, add a sentence, etc.

**Comment by petr-stupka** [comment]
Thanks for your feedback.
May be optional i.e. `Edit this Page in Web Editor` button will be the best solution?

**Comment by chalin** [comment]
> May be optional i.e. `Edit this Page in Web Editor` button will be the best solution?

All buttons are now optional in that they can be disabled. For details see [Disabling links](https://www.docsy.dev/docs/adding-content/repository-links/#disabling-links). So in that sense I wouldn't be against the addition.

(We might debate whether the button should be shown by default or not.)

**Comment by chalin** [comment]
- Pre-0.17.0 triage campaign: #2717
- Answered: all page-meta buttons are configurable; the web-editor flow found no support.
